cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -141,11 +141,16 @@ def test_create_without_description(small_test_vm):
         initialEstimate: 1/4h
         casecomponent: Infra
     """
-    snapshot = new_snapshot(small_test_vm, has_name=False, create_description=False)
-    with pytest.raises(AssertionError):
-        snapshot.create()
-    view = snapshot.parent_vm.create_view(InfraVmSnapshotAddView)
-    view.flash.assert_message('Description is required')
+    if small_test_vm.appliance.version > '5.10':
+        # In 5.10 it's not possible to create a snapshot w/o description,"Create" button is disabled
+        view = navigate_to(small_test_vm, 'SnapshotsAdd')
+        assert view.create.disabled
+    else:
+        snapshot = new_snapshot(small_test_vm, has_name=False, create_description=False)
+        with pytest.raises(AssertionError):
+            snapshot.create()
+        view = snapshot.parent_vm.create_view(InfraVmSnapshotAddView)
```

Comment:
Optional:
I am not tested `shanpshot` as entity but I think it should give you direct `create_view` method... Please check it one time.

Reply:
thanks, I checked and it's not the case. I can access it only through `InfraVm`